### PR TITLE
feat: Create cmdconfig.Binder

### PR DESCRIPTION
**Pull request by michaljurecko**
Jira: https://keboola.atlassian.net/browse/PSGO-439

**Changes:**
- Added binding of os ENVs and ENV files.

-----------

### CODE REVIEW

File: internal/pkg/service/cli/dependencies/dependencies.go [hunk 1]
```diff
@@ -43,6 +44,7 @@ type BaseScope interface {
 	dependencies.BaseScope
 	Fs() filesystem.Fs
 	FsInfo() FsInfo
+	ConfigBinder() *cmdconfig.Binder
 	Dialogs() *dialog.Dialogs
 	Options() *options.Options
```
Comment (michaljurecko) [inline internal/pkg/service/cli/dependencies/dependencies.go:49]: I created the new dependency.

File: internal/pkg/service/cli/cmd/env.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	"context"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+)
+
+func loadEnvFiles(ctx context.Context, logger log.Logger, osEnvs *env.Map, fs filesystem.Fs) *env.Map {
+	// File system basePath = projectDir, so here we are using current/top level dir
+	projectDir := `.` // nolint
```
Comment (michaljurecko) [inline internal/pkg/service/cli/cmd/env.go:13]: Copied from the deprecated `options` package.

The `options` package should be removed at the end.

File: internal/pkg/service/cli/options/options.go [hunk 3]
```diff
@@ -50,9 +50,9 @@ func New() *Options {
 	}
 }
 
-func (o *Options) Load(ctx context.Context, logger log.Logger, osEnvs *env.Map, fs filesystem.Fs, flags *pflag.FlagSet) error {
+func (o *Options) Load(ctx context.Context, _ log.Logger, osEnvs *env.Map, fs filesystem.Fs, flags *pflag.FlagSet) error {
 	// Load ENVs from OS and files
-	o.envs = o.loadEnvFiles(ctx, logger, osEnvs, fs)
+	o.envs = o.loadEnvFiles(ctx, log.NewNopLogger(), osEnvs, fs)
```
Comment (michaljurecko) [inline internal/pkg/service/cli/options/options.go:55]: There is currently ENVs loading twice, by the new code and deprecated `options` package, so I temporary disabled logs from the options package, to fix tests.

File: internal/pkg/service/cli/cmd/ci/workflow/cmd.go [hunk 4]
```diff
@@ -37,7 +36,7 @@ func WorkflowsCommand(p dependencies.Provider) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
 			// Bind flags to struct
 			f := Flags{}
-			if err := configmap.Bind(utils.GetBindConfig(cmd.Flags(), args), &f); err != nil {
+			if err := p.BaseScope().ConfigBinder().Bind(cmd.Flags(), args, &f); err != nil {
 				return err
```
Comment (michaljurecko) [inline internal/pkg/service/cli/cmd/ci/workflow/cmd.go:40]: I updated all commands.

**Review by hosekpeter — APPROVED** [review]
LGTM

**Pull request merged**